modules/Bio/EnsEMBL/StopCodonReadthroughEdit.pm

```diff
@@ -0,0 +1,82 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+  Bio::EnsEMBL::StopCodonReadthroughEdit - Object representing a stop codon readthrough edit in a sequence
+
+=head1 SYNOPSIS
+
+  my $stopcodonrtedit = Bio::EnsEMBL::StopCodonReadthroughEdit->new($coord);
+
+=head1 DESCRIPTION
+
+  A representation of a stop codon readthrough edit in a sequence.
+
+=head1 METHODS
+
+=cut
+
+package Bio::EnsEMBL::StopCodonReadthroughEdit;
+
+use strict;
+use warnings;
+
+use parent qw(Bio::EnsEMBL::SeqEdit);
+
+=head2 new
+
+  Arg [-POSITION]  :
+       int - start and end postion of the stop codon readthrough edit in the sequence
+
+  Example    : $stopcodonrtedit = Bio::EnsEMBL::StopCodonReadthroughEdit->new($posiiton);
+  Description: Creates a new stop codon readthrough edit object
+  Returntype : Bio::EnsEMBL::StopCodonReadthroughEdit
+  Exceptions : none
+  Caller     : general
+  Status     : Stable
+
+=cut
+
+sub new {
+  my ($self, $position) = @_;
+
+  my $class = ref($self) || $self;
+  my $stopcodonrtedit = $class->SUPER::new(
```

Comment:
Historically we have made compound variable names using _ to make them legible, e.g. stop_codon_rt_edit

Reply:
Not just historically, this is explicitly mentioned in our internal style guide.